### ISSUE TITLE
Give Shadowforge to the Horde without war

### ISSUE THREAD

**Issue by ElMariuso** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#402]
> When the Shadow Council moved into Blackrock Mountain, they were confronted by the fire elemental and Dark Iron dwarf minions of Ragnaros. Cho'gall used his fledgling connection to the Old Gods to convince the dwarves and Ragnaros's lieutenants, who also served the Old Gods, to let the Shadow Council stay in the upper reaches of the mountain, Blackrock Spire. If the warlocks remained there, they would not be disturbed by Ragnaros or his followers. Gul'dan was angered that he could not have the whole mountain, but was pleased by Cho'gall's effective diplomacy. However, he was unaware of the true reason it had worked.

Source: https://wowpedia.fandom.com/wiki/Cho%27gall

Half of the kingdom of Shadowforge should be given via event or decision to the Horde by Cho'gall's action. Maybe a bit like the Danelaw negotiation in CK3 vanilla. If the deal is made, the two kingdoms will sign a truce.

I'm saying only half should be given, even though in Chronicles it is marked that the Horde owns all the land for two things:
- So as not to necessarily lose players who would like to play Dark Iron right away.
- To end up with a situation in the region more or less similar to that of World of Warcraft in the event of defeat of the Horde.
If the Horde loses, for example, the clan or vassal that controls this region becomes independent and its titles are not returned to the dark irons.

Basically:
- Burning Steppes to the Dark Horde or the future new Horde, depending on who gets the title.
- Badlands to the Horde
- Searing Gorge to the dark irons.
